fusion_docs/guide/snapshot.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Snapshot for AWS Batch
+description: "Overview of the Fusion Snapshot feature for AWS Batch"
+date: "23 Aug 2024"
+tags: [fusion, storage, compute, file system, snapshot]
+---
+
+Fusion Snapshot enables you to create a checkpoint of a running Nextflow process and restore it on a different machine. Leveraging the Fusion file system, Snapshot enables you to move the task between different instances connected to the same S3 bucket.
```

Comment:
```suggestion
Fusion Snapshots enable you to create a checkpoint of a running Nextflow process and restore it on a different machine. Leveraging the Fusion file system, you can move the task between different instances connected to the same S3 bucket.
```

Reply:
We refer to a "running Nextflow process" and then "task" thereafter. Should we replace "task" with "process"?